opencti-platform/opencti-front/src/private/components/data/connectors/ConnectorWorksErrorLine.tsx

```diff
@@ -58,19 +58,19 @@ const ConnectorWorksErrorLine: FunctionComponent<ConnectorWorksErrorLineProps> =
       <TableRow key={error.rawError.timestamp}>
         <TableCell>{nsdt(error.rawError.timestamp)}</TableCell>
         <TableCell>
-          {error.isParsed ? (
-            <a href={`https://docs.opencti.io/latest/deployment/troubleshooting/#${error.parsedError.category}`} target="_blank" rel="noreferrer">{error.parsedError.category}</a>
+          {error.isParsed && error.parsedError.doc_code ? (
```

Comment:
how will it work for previous errors that don't have a doc_code yet ? will it display only unknown ? the documentation will change and won't be retrocompatible I guess ?